tests/test_json_field_ids.py

```diff
@@ -74,13 +75,26 @@ def check_for_duplicate_field_ids_across_form_jsons():
 
 
 def test_check_form_json_for_duplicate_field_ids():
```

Comment:
Rather than hardcoding and duplicating the english and welsh variants here, could we use parameterisation e.g. https://docs.pytest.org/en/6.2.x/parametrize.html and pass through `en` and `cy` parameters?

Reply:
A neat feature of pytest, thank you 💯